lntest/node.go

```diff
@@ -475,24 +480,31 @@ func (hn *HarnessNode) start(lndBinary string, lndError chan<- error) error {
 	// If the logoutput flag is passed, redirect output from the nodes to
 	// log files.
 	if *logOutput {
-		fileName := fmt.Sprintf("output-%d-%s-%s.log", hn.NodeID,
-			hn.Cfg.Name, hex.EncodeToString(hn.PubKey[:logPubKeyBytes]))
+		fileName := fmt.Sprintf("%03d-%v-%s-%s.log", hn.NodeID,
```

Comment:
This needs a change in the root `.gitignore` file.

Reply:
done